src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -205,6 +328,46 @@ IN_PROCESS_APPLICATION::~IN_PROCESS_APPLICATION()
     Recycle();
 }
 
+REQUEST_NOTIFICATION_STATUS
+IN_PROCESS_APPLICATION::OnAsyncCompletion(
```

Comment:
It does not make sense to me to have this member method. You can directly call it in FORWARDING_HANDLER::OnAsyncCompletion

Reply:
The handler is unique to the application though, so either we need to expose it to the forwarding handler or do something else.